### DISCUSSION TITLE
mpremote failed zu access

### DISCUSSION THREAD

**Discussion by hotschy** [micropython/micropython#13581] [RP2040 / Pico] [answered]
I want to get mpremote access to my pico connected via usb.
I get this answer:

hotsch@RB-04: $ python3 -m mpremote connect list
/dev/ttyACM0 e6633861a3543a38 2e8a:0005 MicroPython Board in FS mode
/dev/ttyS4 None 0000:0000 None None
hotsch@RB-04: $ python3 -m mpremote connect port:/dev/ttyACM0
**failed to access /dev/ttyACM0 (it may be in use by another program)**

(Linux Mint 21.1 Vera)

I have no idea how to proceed. Can anyone help me?

**Comment by andrewleech** [comment]
This is a common issue on linux, typically it's because of `modemmanager` is installed and running by default which takes over pretty much any generic usb serial just in case it might be a modem of some kind... https://github.com/RfidResearchGroup/proxmark3/blob/master/doc/md/Installation_Instructions/ModemManager-Must-Be-Discarded.md

**Reply by hotschy** [reply]
Great Andrew,
thank you solved!

**Comment by GitHubsSilverBullet** [comment]
> I want to get mpremote access to my pico connected via usb. I get this answer:
> 
> hotsch@RB-04: $ python3 -m mpremote connect list /dev/ttyACM0 e6633861a3543a38 2e8a:0005 MicroPython Board in FS mode /dev/ttyS4 None 0000:0000 None None hotsch@RB-04: $ python3 -m mpremote connect port:/dev/ttyACM0 **failed to access /dev/ttyACM0 (it may be in use by another program)**
> 
> (Linux Mint 21.1 Vera)
> 
> I have no idea how to proceed. Can anyone help me?

Make sure that you're a member of the dialout group.

**Comment by hotschy** [comment]
Hey Mark,
Thanks for the tip!
Unfortunately that's not it, I'm a member of the dialout group

**Reply by robert-hh** [reply, marked as answer]
See the comment of @andrewleech above about the modemmanager. That's the problem I had with a debian system. On a Ubunti system I had problems with a braille reader service, which had to be removed. See discussion #13418. Even if that only affects /dev/ttyUSBx, in that discussion there is also a hint at the very end by Jimmo to remove the modemmanager.

**Reply by hotschy** [reply]
Thank you, I overlooked this
done!
